app/src/app.js

```diff
@@ -3,8 +3,37 @@ const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
 const postValidation = require('./validation/post');
+const accounts = require ('./model/accounts');
+const password = require ('./security/password');
+const passValidation = require ('./validation/passwordValidation');
 app.use(bodyParser.json());
 
+app.post('/register', async (req, res) => {
+    let newUser = new accounts.UserEntity(null, req.body.username, req.body.password);
+
+    let errorsArray = await passValidation.validatePass(newUser);
```

Comment:
W kodzie nie powinno się robić skrótów w stylu `pass` zamiast `password`. Po pierwsze piszesz to raz a czytasz sto razy, więc ważniejsza jest czytelność. Po drugie "pass" znaczy "przejść", "zatwierdzić" więc nijak się to ma do tego co robi kod 

Dwa funkcja `validatePass(word)` sugeruje że to walidator hasła, a nie użytkownika. No a przekazujesz do środka użytkownika

Reply:
poprawione